src/Repositories/ExtensionRepository.php

```diff
@@ -136,8 +176,6 @@ public function createExtension(array $apiPackage)
 
         $this->refreshInstalledExtension($extension);
 
-        $this->flushCacheKey('flagrow.io.search.list');
```

Comment:
@luceos this breaks the cache because it's called everytime we load an extension from the cache. Any idea why it was there ?